### DISCUSSION TITLE
Inquiry Regarding ArangoDB License Change and Its Impact on Spline

### DISCUSSION THREAD

**Discussion by harikrishna553** [AbsaOSS/spline#1357] [General]
Hi Spline Team,

We noticed that the Spline project uses ArangoDB. Recently, ArangoDB's license changed from Apache to BSL (Business Source License).

Given this change, we wanted to inquire about the Spline team's perspective on how this new license might impact the use of ArangoDB within the Spline project. Specifically, are there any anticipated effects or considerations we should be aware of if we continue to use Spline with ArangoDB in its new licensing model?

Any guidance or insights you could provide would be greatly appreciated.

Thank you for your support and for the great work on Spline!

References:
https://arangodb.com/2024/02/update-evolving-arangodbs-licensing-model-for-a-sustainable-future/

**Comment by wajda** [comment]
So far we do not have any plans to take any actions in response to ArangoDb licence change. From the perspective of Spline as open-source project, the license change does not affect us. We do not ship or embed ArangoDb in any Spline component, except for Arango DB Java driver, that is still distributed under Apache-2.0 license. The responsibility of being in compliance with the ArangoDb server license fully lies with the customer.

**Reply by harikrishna553** [reply]
Thanks for the information
